### PR TITLE
RATEST-250: Restore User account workflow test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket [ID](https://issues.openmrs.org/browse/RATEST-250)

Description -> Restoring user account test into master after fixing ci build plan.

### CODE REVIEW

File: src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
```
Comment (sherrif10) [inline src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java:2]:
```suggestion
 * This Source Code Form is subject to the terms of the Mozilla Public License,
```
/**
 * This Source Code Form is subject to the terms of the Mozilla Public License,
 * v. 2.0. If a copy of the MPL was not distributed with this file, You can
 * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
 * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
 * 
 * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
 * graphic logo is a trademark of OpenMRS Inc.
 */

Make  headerPages have something info like this

Comment (kdaud) [reply src/test/java/org/openmrs/contrib/qaframework/automation/UserAccountSteps.java:2]: @sherrif10 could there be something am missing!

Currently I have;
```js
/**
 * This Source Code Form is subject to the terms of the Mozilla Public License,
 * v. 2.0. If a copy of the MPL was not distributed with this file, You can
 * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
 * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
 * <p>
 * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
 * graphic logo is a trademark of OpenMRS Inc.
 */
```

**Review by sherrif10 — APPROVED** [review]
Thanks @kdaud

**Comment by sherrif10** [comment]
@dkayiwa  this looks good for merge, Lets keep an eye on the ci build again

**Pull request merged**